### ISSUE TITLE
Wrong Price calculated 

### ISSUE THREAD

**Issue by AkashShrestha31** [webkul/hotelcommerce#1314]
When i allocated date 11/26/2024 - 11/27/2024
 its cost  is 
<img width="431" alt="image" src="https://github.com/user-attachments/assets/21a65d83-1087-4aaf-89b6-82f994bae669">
and after i update checkout date 11/26/2024 - 11/28/2024 but still it is saying 
<img width="423" alt="image" src="https://github.com/user-attachments/assets/f0b60e4f-56de-4891-bc03-7ae31a8e5185">
 and when user pay 3000 it says this 
<img width="735" alt="image" src="https://github.com/user-attachments/assets/7a19b383-0386-4f97-a36b-f855f950fa56">
and in dashboard it shows 
<img width="220" alt="image" src="https://github.com/user-attachments/assets/800f595e-5836-41e5-ab9b-a683cdcd280c">
Do anyone have solution?

**Comment by aman-webkul** [comment]
Yes, the folowing PR [#1308 ](https://github.com/Qloapps/QloApps/pull/1308) will fix the issue

**Comment by AkashShrestha31** [comment]
Thanks. hope pr will be merge soon.
